src/dict/dictionary.cc

```diff
@@ -509,26 +509,16 @@ string makeDictionaryId( vector< string > const & dictionaryFiles ) noexcept
 {
   std::vector< string > sortedList;
 
-  if ( Config::isPortableVersion() ) {
-    // For portable version, we use relative paths
+  
+      // use filename to unify the calculation step
     sortedList.reserve( dictionaryFiles.size() );
```

Comment:
Nitpicking: 

This can be combined into a single line now.

`std::vector< string > sortedList(dictionaryFiles.size())`;